Create meeting recording policy (#2819)
open-telemetry/community: policies/third-party-meeting-recordings.md

Reviewer: These should likely be moved to the new `policies/` folder as well in a follow-up:

https://github.com/open-telemetry/community/blob/main/marketing-guidelines.md
https://github.com/open-telemetry/community/blob/main/social-media-guide.md
perhaps also
https://github.com/open-telemetry/community/blob/main/project-management.md

Reviewer 2: will create a follow up issue/PR

# Third-party meeting recording policy

OpenTelemetry hosted meetings are recorded and transcribed by Zoom and made available for the convenience of the community ([details here](../docs/how-meeting-recordings-upload-works.md)).

> Third party recording tools, note-taking apps, and other tools which record, transcribe, or otherwise process OpenTelemetry Zoom meetings are prohibited.

Reviewer 3: I haven't heard objections to local recording tools that someone may use while they themselves are attending the meeting.
```suggestion
Third party recording tools that join Zoom meetings are prohibited.
```

Reviewer 4: Is this now solved?

> Maintainers should remove and block such tools using the process described in [Zoom abuse prevention](../docs/how-to-handle-public-calendar.md#zoom-abuse-prevention).

Reviewer 3: ```suggestion
Maintainers may remove and block such tools using the process described in [Zoom abuse prevention](../docs/how-to-handle-public-calendar.md#zoom-abuse-prevention).
```
